tests/lib/tools/setup_nested_hybrid_system.sh

```diff
@@ -164,10 +167,15 @@ fi
 EOF
     remote.exec "chmod +x /home/user1/mk-classic-rootfs-wrapper.sh"
 
-    remote.exec "sudo muinstaller \
-        -label ${label} \
-        -device ${install_disk} \
-        -rootfs-creator /home/user1/mk-classic-rootfs-wrapper.sh"
+    muinstaller_args=()
+    muinstaller_args+=("-label $label")
+    muinstaller_args+=("-device $install_disk")
+    muinstaller_args+=("-rootfs-creator /home/user1/mk-classic-rootfs-wrapper.sh")
```

Comment:
Shouldn't each argument be separate? Like `args+=("-label" "$label")`, etc. I think that in the end is not relevant as `remote.exec` somewhat is possibly splitting arguments, but still.

Reply:
Thanks for catching this, updated